pkgs/development/python-modules/shiboken6/default.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, fetchurl
+, llvmPackages
+, python
+, qt6
+, cmake
+, autoPatchelfHook
+, stdenv
+, libxcrypt
+}:
+
+llvmPackages.stdenv.mkDerivation rec {
+  pname = "shiboken6";
+  version = "6.5.0";
+
+  src = fetchurl {
+    # https://download.qt.io/official_releases/QtForPython/shiboken6/
+    url = "https://download.qt.io/official_releases/QtForPython/shiboken6/PySide6-${version}-src/pyside-setup-everywhere-src-${version}.tar.xz";
+    sha256 = "sha256-bvU7KRJyZ+OBkX5vk5nOdg7cBkTNWDGYix3nLJ1YOrQ=";
+  };
+
+  sourceRoot = "pyside-setup-everywhere-src-${lib.versions.majorMinor version}/sources/${pname}";
+
+  patches = [
+    ./fix-include-qt-headers.patch
```

Comment:
Is this patch upstreamed / upstreamable?

Reply:
Just had a look into this, I don't think it's upstreamable because the logic involves matching paths against the Nix store. We're already using a similar patch for `shiboken2` already.